.github/workflows/test-and-lint.yml

```diff
@@ -0,0 +1,30 @@
+name: Test and lint
+
+on: [push]
```

Comment:
not sure it will be triggered for external contributors 😞 I think you'll need to add the `pull_request` hook.

Reply:
Indeed, good catch 👍 I'm going to change it with:
```
on: [push, pull_request]
```

Reply:
I just noticed that when it's your pull request, the process will run twice 😅 Maybe just using pull_request would work?

Reply:
or maybe something like that:
```
on:
  pull_request
  push:
    branches:
      - master
```

I think it will trigger on all pull_request or when someone push on master.

Reply:
I confirm this one is working